cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/ConcurrentStreamWriter.java

```diff
@@ -321,14 +349,21 @@ public void deleted(Id.Stream streamId) {
      * @return A {@link FileWriter} for writing {@link StreamEvent} to the given stream
      * @throws IOException if failed to create the file writer
      */
-    FileWriter<StreamEvent> create(Id.Stream streamId) throws Exception {
+    private FileWriter<StreamEvent> create(Id.Stream streamId) throws IOException {
```

Comment:
But I don't see code change on the caller of getFileWriter. The refresh method can throw Exception. I don't see why it has to limit to IOException. The concept of Refreshable is pretty generic